operational/stranded_servers/Stranded_Servers.pt

```diff
@@ -0,0 +1,191 @@
+  actions   "rs_cm.show_sensitive","rs_cm.index_sensitive"
+  resources "rs_cm.credentials"
+end
+
+###############################################################################
+# User inputs
+###############################################################################
+
+parameter "param_email" do
+  type "list"
+  label "Email addresses to notify"
+  description "Email addresses of the recipients you wish to notify when new incidents are created"
+end
+
+parameter "param_exclude_tags" do
+  type "list"
+  label "Tags to ignore"
+  description "List of tags that will exclude Servers from being evaluated by this policy. Multiple tags are evaluated as an 'OR' condition. Tag must be of the format 'namespace:predicate=value'. Example: 'rs_agent:type=right_link_lite,rs_monitoring:state=auth'"
+end
+
+###############################################################################
+# Authentication
+###############################################################################
+
+auth "auth_rs", type: "rightscale"
+
+###############################################################################
+# Datasources
+###############################################################################
+
+datasource "ds_list_stranded_instances" do
+  request do
+    run_script $js_list_stranded_instances, rs_cm_host, rs_project_id, $param_exclude_tags
+  end
+  result do
+    encoding "json"
+    collect jmes_path(response, "[*]") do
+      field "cloud_name", jmes_path(col_item, "links.cloud.name")
+      field "cloud_type", jmes_path(col_item, "links.cloud.cloud_type")
+      field "name", jmes_path(col_item, "name")
+      field "resource_uid", jmes_path(col_item, "resource_uid")
+      field "updated_at", jmes_path(col_item, "links.deployment.timestamps.updated_at")
+      field "locked", jmes_path(col_item, "locked")
+      field "href", jmes_path(col_item, "href")
+      field "tags", jmes_path(col_item, "tags")
+      field "state", jmes_path(col_item, "state")
+    end
+  end
+end
+
+datasource "ds_get_stranded_instances" do
+  run_script $js_get_stranded_instance, $ds_list_stranded_instances, $param_exclude_tags
+end
+
+###############################################################################
+# Script
+###############################################################################
+
+script "js_list_stranded_instances", type: "javascript" do
+  result "request"
+  parameters "rs_cm_host","rs_project_id","param_exclude_tags"
+  code <<-EOS
+    var filtered_tags = ""
+    if ( param_exclude_tags.length > 0 ){
+      var filtered_tags = "&tag!=" + param_exclude_tags.join(",");
+    }
+
+    request = {
+      "auth": "auth_rs",
+      "verb": "GET",
+      "host": rs_cm_host,
+      "path": "/api/instances",
+      "headers": {
+        "X-Api-Version": "1.6",
+        "X-Account": \""+rs_project_id+\""
+      },
+      "query_params": {
+        "view": "full",
+        "filter": "state=stranded in booting" + filtered_tags
```

Comment:
the state can be just stranded as well, not just stranded in booting.

Reply:
Thanks, I have updated.